src/op_mode/generate_firewall_rule-resequence.py

```diff
@@ -116,9 +116,18 @@ def convert_rule_keys_to_int(config_dict):
         print('Firewall is not configured')
         exit(1)
 
-    #config_dict =  config.get_config_dict('firewall')
     config_dict = config.get_config_dict('firewall')
 
+    # Remove global-options, group and flowtable as they don't need sequencing
+    if 'global-options' in config_dict['firewall']:
```

Comment:
This (and in the `del` statement) should be be the node mangled value 'global_options' (dashes mangled to underscores)